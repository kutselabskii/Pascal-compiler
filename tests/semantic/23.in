program abc;

var a: real;

begin
	while (a) do
	begin
		a := a + 1;
	end;
end.