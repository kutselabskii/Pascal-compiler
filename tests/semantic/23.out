Line 7 : Col 13 : Semantic error :: While condition should be 'integer' value
